product_docs/docs/biganimal/release/using_cluster/04_fault_injection_testing_your_cluster/fault_injection_testing.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: "Fault injection testing"
+---
+
+You can test the fault tolerance of your cluster by deleting a VM in order to inject a fault. Once a VM is deleted, you can monitor 
+the availability and recovery of the cluster.
+
+## Requirement
+
+Ensure you meet the following requirements before using fault injection testing:
+
++ You have connected your BigAnimal cloud account with your Azure subscription.  See [Setting up your Azure Marketplace account](/biganimal/latest/getting_started/02_azure_market_setup/) for more information.
++ You should have permissions in your Azure subscription to view and delete VMs.
++ You have **pgd cli** installed.  See [Installing PGD CLI](/pgd/latest/cli/installing_cli/#) for more information. 
++ You have created a **pgd-cli-config.yml** file in your home directory.  See [Configuring PGD CLI](/pgd/latest/cli/configuring_cli/) for more information.
+
+## Fault injection testing steps
+
+Fault injection testing consists of the following steps:
+
+1. Verifying Cluster Health
+2. Determining the write leader node for your cluster
+3. Deleting a write leader node from your cluster
+4. Monitoring cluster health
+&nbsp;
+&nbsp;
+#### Verifying Cluster Health
```

Comment:
Should be h3 below h2 (`##` -> `###`). See https://www.enterprisedb.com/docs/community/contributing/styleguide/#headings